test/e2e/httpproxy/008_tcproute_https_termination_test.go

```diff
@@ -29,7 +28,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func testTCPRouteHTTPSTermination(t *testing.T, fx *e2e.Framework) {
+func testTCPRouteHTTPSTermination(fx *e2e.Framework) {
+	t := fx.T()
```

Comment:
maybe we could do some embedding of test function tricks to make this cleaner? can do in another PR tbh